js/index.js

```diff
@@ -1 +1,200 @@
+    funBusHeading[3].style.fontSize = "16px";
+})
+
+
+//// Blog link
+
+funBusHeading[2].addEventListener("mouseover", e => {
+    funBusHeading[2].style.color = "red";
+    funBusHeading[2].style.background = "green";
+    funBusHeading[2].style.padding = "3%";
+    funBusHeading[2].style.borderRadius = "20%";
+    funBusHeading[2].style.boxShadow = "2px 17px 35px 1px #8E0000";
+    funBusHeading[0].style.fontSize = "10px";
+    funBusHeading[1].style.fontSize = "10px";
+    funBusHeading[3].style.fontSize = "10px";
+})
+
+funBusHeading[2].addEventListener("mouseout", e => {
+    funBusHeading[2].style.background = "white";
+    funBusHeading[2].style.color = "black";
+    funBusHeading[2].style.padding = "0%";
+    funBusHeading[2].style.boxShadow = "none";
+    funBusHeading[0].style.fontSize = "16px";
+    funBusHeading[1].style.fontSize = "16px";
+    funBusHeading[3].style.fontSize = "16px";
+})
+
+//// Contact link
+
+funBusHeading[3].addEventListener("mouseover", e => {
+    funBusHeading[3].style.color = "red";
+    funBusHeading[3].style.background = "green";
+    funBusHeading[3].style.padding = "3%";
+    funBusHeading[3].style.borderRadius = "20%";
+    funBusHeading[3].style.boxShadow = "2px 17px 35px 1px #8E0000";
+    funBusHeading[0].style.fontSize = "10px";
+    funBusHeading[1].style.fontSize = "10px";
+    funBusHeading[2].style.fontSize = "10px";
+})
+
+funBusHeading[3].addEventListener("mouseout", e => {
+    funBusHeading[3].style.background = "white";
+    funBusHeading[3].style.color = "black";
+    funBusHeading[3].style.padding = "0%";
+    funBusHeading[3].style.boxShadow = "none";
+    funBusHeading[0].style.fontSize = "16px";
+    funBusHeading[1].style.fontSize = "16px";
+    funBusHeading[2].style.fontSize = "16px";
+})
+
+funBusHeading[3].addEventListener("click", e => {
+    playSound("Krueger.mp3");
+    e.preventDefault();
+    e.stopPropagation();
+});
+
+/// Images
+
+funBusImages[0].addEventListener("mouseover", e => {
+    funBusImages[0].src = "https://pa1.narvii.com/6846/79706defae23d7cdb1103648171f261c61375ed8_hq.gif";
+    funBusImages[0].align ="right";
+    playSound("Freddie.mp3")
+})
+
+funBusImages[1].addEventListener("mouseover", e => {
+    funBusImages[1].src = "https://j.gifs.com/864LV5.gif";
+})
+
+funBusImages[1].addEventListener("mouseout", e => {
+    document.body.style.backgroundColor = "black";
+    document.body.style.color = "red";
+})
+
+funBusImages[2].addEventListener("mouseover", e => {
+    funBusImages[2].src = "https://j.gifs.com/864LQW.gif";
+})
+
+funBusImages[3].addEventListener("mouseover", e => {
+    funBusImages[3].src = "https://i0.wp.com/riversofgrue.com/wp-content/uploads/2013/06/noesf.gif?resize=450%2C253&ssl=1";
+    playSound("Hahahahaha.mp3")
+})
+
+////// destination containers
+
+// destinationContainer[0].addEventListener("mouseover", () => {
+//     destinationContainer[0].style.color = "green";
+// })
+
+
+
+////// buttons
```

Comment:
Same for the buttons, you can just use `.forEach` and apply to all of them when you `.querySelectorAll`